docs/source/reservations/existing-machines.rst

```diff
@@ -70,22 +68,22 @@ Deploying SkyPilot
 
    In this example, the first node (``192.168.1.1``) has port 6443 open and will be used as the head node.
 
-2. Run ``sky local up`` and pass the ``ips.txt`` file, SSH username, and SSH keys as arguments:
+2. Run ``sky local up`` and pass the ``ips.txt`` file, SSH username, and SSH key as arguments:
 
    .. code-block:: bash
 
       IP_FILE=ips.txt
-      SSH_USERNAME=username
+      SSH_USER=username
       SSH_KEY=path/to/ssh/key
-      sky local up --ip $IP_FILE --username $SSH_USERNAME --key-path $SSH_KEY
+      sky local up --ip $IP_FILE --ssh-user SSH_USER --ssh-key-path $SSH_KEY
```

Comment:
Is it `--ip` or `--ips` (L86)? Latter is more intuitive. (Or --ipfile like DeepSpeed's --hostfile.)

Reply:
ah good catch - it is indeed `--ips`, this typo was a remnant from the old deploy.sh.